examples/tracing/otlp-collector/llm.py

```diff
@@ -0,0 +1,88 @@
+import os
+
```

Comment:
maybe add this to this PR https://github.com/microsoft/promptflow/pull/1741

We can we merge them together